modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java

```diff
@@ -97,23 +97,39 @@ public void testFaultManagement() throws Exception {
         assertEquals(1, historicProcessInstanceWithExpectedFault.size());
         assertEquals("theEndWithError", historicProcessInstanceWithExpectedFault.get(0).getEndActivityId());
 
-        // Runtime exception occurring during processing of the web-service, so not catched in the process definition
+        // RuntimException occurring during processing of the web-service, so not catched in the process definition. The RuntimeException
+        // is embedded as an unexpected fault at web-service server side
         webServiceMock.setTo(123456);
         try {
             runtimeService.startProcessInstanceByKey("webServiceInvocation");
         } catch (ActivitiException e) {
+            assertFalse("Exception processed as Business fault", e instanceof BpmnError);
             assertTrue(e.getCause() instanceof SoapFault);
         }
 
-        // Unexpected fault invoking the web-service, so not catched in the process definition
+        // Unexpected fault at ws-client side invoking the web-service, so not catched in the process definition
         server.stop();
         try {
             runtimeService.startProcessInstanceByKey("webServiceInvocation");
         } catch (ActivitiException e) {
+            assertFalse("Exception processed as Business fault", e instanceof BpmnError);
             assertTrue(e.getCause() instanceof Fault);
         } finally {
             server.start();
         }
     }
 
+    @Deployment
+    public void testFaultManagementREatWSClientLevel() throws Exception {
+
+        // RuntimException occurring during processing of the web-service at ws-client side, so not catched in the
```

Comment:
Should read:
RuntimeException occurring during processing of the web-service at ws-client side, so not caught